### DISCUSSION TITLE
Choropleth Any Map Of Earth Map

### DISCUSSION THREAD

**Discussion by aufukgarip** [sgratzl/chartjs-chart-geo#269] [Ideas]
Choropleth Any Map Of Earth Map is possible by using this technology instead of us choropleth map, if it is, How? Thanks!

**Comment by sgratzl** [comment]
what do you, the second example and screenshot in the README is showing the whole earth?

**Comment by aufukgarip** [comment]
Ok sorry but i couldnt clarify enough my opinion i wouldnt like whole earth, only a specific country as a chropoleth e.g Turkey map as chropoleth(like screenshot1 in readme)by using this technology.Thanks.

**Reply by sgratzl** [reply]
this plugin can render any topojson file you give it. It comes down on how to get this file and what is the best projection on how to present it. For both I'm not an expert.

**Reply by abdullahufuk** [reply]
The goal to open this discussion is obtaining a good topojson file for all countries seperately for using it on chropleth map but as you mentioned you'r not expert.Anyway thanks for your reply!
